specification/schemas/responses/MessageResponse.yaml

```diff
@@ -0,0 +1,186 @@
+title: MessageResponse
+type: object
+additionalProperties: false
+properties:
+  data:
+    type: object
+    additionalProperties: false
+    properties:
+      type:
+        $ref: ../enums/TypeMessage.yaml
+      id:
+        $ref: ../types/KSUID.yaml
+        description: Identifier for this Message.
```

Comment:
Missing the example on this, needs to be the same value as in the `self` url below: `0ujsszwN8NRY24YaXiTIE2VWDTS `

Reply:
Added example